pycipapi/rest_client.py

```diff
@@ -175,8 +175,8 @@ def _verify_response(self, response, method=None, **kwargs):
         )
         if response.status_code not in (200, 203, 206, 201):
             logging.error(response.content)
-            # first 403 renews the token, second 403 in a row fails
-            if response.status_code == 403 and not self.renewed_token:
+            # first 401/403 renews the token, second 401/403 in a row fails
+            if response.status_code in (401, 403) and not self.renewed_token:
```

Comment:
I'll approve this PR, 

But we shouldn't retry 403, this seems like an API bug. 403 is forbidden - i.e. user logged in, but doesn't have permissions to view the resource, i.e. getting the same token shouldn't help in this case as the same permissions stay.

Reply:
Thanks. Yeah I was wondering if the API used to return 403 on token expiry, but it's been updated to return 401? I just left the 403 there as well to avoid breaking anything, but probably it could be removed.